.github/workflows/build.yaml

```diff
@@ -16,8 +18,15 @@ env:
   MAINTAINER_EMAIL: Armand.Meppa@adorsys.com
```

Comment:
Why the need of the maintenaner email?

Reply:
It was to create a *.deb* file. But with the new way, it's no more useful